server/app/assets/javascripts/azure_delete.ts

```diff
@@ -3,8 +3,12 @@
  */
 class AzureDeleteController {
   private static FILEUPLOAD_DELETE_ID = 'fileupload-delete-button'
+  private static AZURE_UPLOAD = 'azure-upload'
```

Comment:
suggestion (blocking): this value is being used as a query selector, but is missing a `.` at the beginning (assuming this is querying for an HTML class). Please add a test that fails when this selector is incorrect.

Reply:
suggestion (blocking): please rename to `AZURE_UPLOAD_SELECTOR`

Reply:
I can't add tests at the moment as we don't have infra for TS tests in server folder. I'll need to implement it first. Will do it in parallel.